tests/eosio.system_tester.hpp

```diff
@@ -752,34 +752,15 @@ class eosio_system_tester : public TESTER {
       base_tester::push_action(contract, N(create), contract, act );
    }
 
-   void issue( name to, const asset& amount, name manager = config::system_account_name ) {
-      signed_transaction trx;
-      trx.actions.emplace_back(
-         get_action( N(eosio.token), N(issue),
-                     vector<permission_level>{{manager, config::active_name}},
-                     mutable_variant_object()
-                        ("to",       manager )
-                        ("quantity", amount )
-                        ("memo",     "")
-         )
-      );
-      if( to != manager ) {
-         trx.actions.emplace_back(
-            get_action( N(eosio.token), N(transfer),
-                        vector<permission_level>{{manager, config::active_name}},
-                        mutable_variant_object()
-                           ("from",     manager)
-                           ("to",       to )
-                           ("quantity", amount )
-                           ("memo",     "")
-            )
-         );
-      }
-      set_transaction_headers( trx );
-      trx.sign( get_private_key( manager, "active" ), control->get_chain_id()  );
-      push_transaction( trx );
+   void issue( const name& to, const asset& amount, const name& manager = config::system_account_name ) {
```

Comment:
Remove `to` argument. Set the `to` field to `manager`.

Reply:
Done.